src/Effects.js

```diff
@@ -479,6 +537,36 @@ module.exports = class Effects {
       }
     };
 
+    this.lasers = {
+      laser: [],
+      draw: function () {
+        p5.background('black');
+        let laser = {
+          w: 200,
+          x: 200,
+          y: 1700,
+          z: 400,
+          color: randomColor(255,255,255),
+        };
+        this.laser.push(laser);
+        p5.stroke('white');
+        p5.line(0,200,400,200);
+        this.laser.forEach(laser => {
+          p5.push();
+          p5.translate(this.laser.w, this.laser.x, this.laser.y, this.laser.z);
```

Comment:
What does this do? [`p5.translate`](https://p5js.org/reference/#/p5/translate) is only documented as taking three arguments.

Reply:
@islemaster cleaning these up as part of later merge commit